optimum/neuron/modeling_diffusion.py

```diff
@@ -698,6 +699,7 @@ def _export(
             save_dir = TemporaryDirectory()
             save_dir_path = Path(save_dir.name)
             # 1. Fetch all model configs
+            input_shapes_copy = copy.deepcopy(input_shapes)
```

Comment:
nit: maybe just shadow `input_shapes` and do:
```suggestion
            input_shapes = copy.deepcopy(input_shapes)
```

Reply:
this will shadow it but before it's actually updated in the call below, so the problem won't be fixed, no ?

Reply:
Ok, no worries then, let's keep it like that.